setup.cfg

```diff
@@ -16,5 +16,6 @@ ignore =
 
 [flake8]
 max-line-length = 90
-#  W503 requires that binary operators be at the end, not start, of lines. Erik doesn't like it.
-ignore = W503
+#  W503 requires that binary operators be at the end, not start, of lines. Erik doesn't like it.\
```

Comment:
what's with the trailing \\ here?

Reply:
me hitting my keyboard wrong with my dumb monkey hands (fixed)